markdown-link-check

```diff
@@ -2,326 +2,8 @@
 
 'use strict';
```

Comment:
The mutation testing could not deal with this file not having a `.js` extension, so I moved it to `src/markdown-link-check.js` and this file just executes that code now.

Also removed some unused variables etc. in the new file now that it's being checked by the linter because it has a `.js` extension.